tests/Makefile

```
@@ -0,0 +1,133 @@
+TOOL_PATH?=sfpi/compiler/bin
+BUILD_DIR ?=build
+
+RMDIR := rm -rf
+
+# Tool paths
+GXX=$(TOOL_PATH)/riscv32-unknown-elf-g++
+OBJDUMP=$(TOOL_PATH)/riscv32-unknown-elf-objdump
+OBJCOPY=$(TOOL_PATH)/riscv32-unknown-elf-objcopy
+READELF=$(TOOL_PATH)/riscv32-unknown-elf-readelf
+
+# GCC options
+OPTIONS_ALL=-O3 -mabi=ilp32 -std=c++20 -g -ffast-math 
```

```suggestion
OPTIONS_ALL=-O3 -mabi=ilp32 -std=c++20 -g -ffast-math -fprofile-arcs -ftest-coverage
```
To enable coverage. Let's see if that's enough for now.

---

Have You found out which compile flags lead to overflow in sections and which are ok to be added ?